pages/intro/[[...page]].jsx

```diff
@@ -45,6 +52,43 @@ export async function getStaticProps({ params }) {
       const filepath = path.replace('content/', '')
       return `https://github.com/hashicorp/${PRODUCT.slug}/blob/main/website/${filepath}`
     },
+    remarkPlugins: [
+      () => {
+        return async function transform(tree) {
+          const promises = []
+
+          visit(tree, 'image', (node, index, parent) => {
+            if (!node.url.startsWith('http')) {
+              const promise = (async () => {
+                try {
+                  const relativePath = `./public${node.url}`
+                  const { width, height } = await sizeOf(relativePath)
+                  const props = [
+                    ['src', `"${node.url}"`],
```

Comment:
💭 pass `alt` along too?

Reply:
whoops, totally forgot to stage that change!